massgov/mayflower-patternlab
# DP-8890: Always print stickTOC expander to let JS handle visibility

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

This updates the sticky TOC to always print the "Show More" expander markup.  JavaScript will control the visibility of the expander based on the number of sections.

## Related Issue / Ticket

- [DP-8890](https://jira.state.ma.us/browse/DP-8890)

## Steps to Test


1. Checkout the branch
1. Build mayflower locally
1. Confirm that the sticky [TOC organism (by template)](http://localhost:3001/?p=organisms-sticky-toc) has not changed from the [current version in Mayflower](https://mayflower.digital.mass.gov/?p=organisms-sticky-toc)

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
